R/pmcmc.R

```diff
@@ -86,6 +90,9 @@ pmcmc <- function(data,
                   n_mcmc, 
                   model_params = NULL,
                   model_start_date = "2020-02-02",
+                  generate_beta_func = function(beta_start, start_date) 
```

Comment:
also here, can be done more simply.  However if we're repeating it everywhere should we look at lumping this into the generic "model" object perhaps?